x-pack/plugins/ml/public/application/components/data_grid/feature_importance/use_classification_path_data.tsx

```diff
@@ -49,30 +56,43 @@ export const getStringBasedClassName = (v: string | boolean | undefined | number
   return v;
 };
 
+export const formatValue = (number: number, precision = 3, fractionDigits = 1): string => {
+  if (Math.abs(number) < 10) {
+    return Number(number.toPrecision(precision)).toString();
```

Comment:
According to the [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Number/toPrecision) `toPrecision` returns a string, so there is no need to use `Number` and `toString`.

Reply:
This function is being replaced with the already existing `formatSingleValue` so I'm going to resolve this for now. But we cast it inside Number() because if not the result will be shown in scientific notation.